### ISSUE TITLE
Text in the cortex plugin should be bolder it is harder to read.

### ISSUE THREAD

**Issue by riginoommen** [cortexapps/backstage-plugin#161]
Text in the cortex frontend plugin is very harder to read. in consideration with accessibility. the text colour should be bold 

![Screenshot 2024-04-16 at 2 20 02 PM](https://github.com/cortexapps/backstage-plugin/assets/3840244/5063be7b-edde-40d3-a100-c83f071f505d)

**Comment by starryeyez024** [comment]
This problem exists on individual services (as shown above) as well as on the rules for individual scorecards. The colors should probably be inverted, where the rule description text is black, and the code is gray.
 
Current:
![image](https://github.com/cortexapps/backstage-plugin/assets/456863/64316770-cece-4676-a0e3-379a939f4a19)

Suggested:
![image](https://github.com/cortexapps/backstage-plugin/assets/456863/386e4876-a0a5-4fde-a6d2-f4e6cb913323)

**Comment by starryeyez024** [comment]
Currently we are on Cortex 0.0.293 , Backstage Version 1.21.1 (but these screenshots are just from Backstage).
